iso/8.2/.treeinfo

```diff
@@ -1,17 +1,17 @@
 [platform]
 name = XCP
-version = 3.2.0
+version = 3.2.1
 
 [branding]
 name = XCP-ng
-version = 8.2.0
+version = 8.2
 
 [build]
-number = release/stockholm/master/7
+#number = release/yangtze/master/58
```

Comment:
Remains the question of how to differenciate a test build from an official release, and differenciate two subsequent builds. The timestamp will help to some extent, but if we want to go further we could maybe append stuff to either the build number (if diverging from the actual product build number which is in the `xcp-ng-release` RPM is not a problem) or the general name, based on the repo config used and the date.

For example, if we use a `8.3:ci` config, then `general.name` could become `XCP-ng-8.3-ci-20230517`.

Only builds that correspond to official releases would not bear this suffix, so `general.name` would remain `XCP-ng-8.3` for the official release. 

Ideally we'd use the same suffix for the produced ISOs, and this would be a `--name-suffix` setting who could have one the following values:
* `auto`: let the script build the suffix (default?)
* `none`: no suffix at all (official releases)
* anything else: use this input as the suffix. If I want to suffix it `-stormi-test-build-for-I225-V`, I can.

Reply:
If we start adding `%d%m%y` in filename, output file is not clobbered by default any more, so we'll likely want to have `%H%M%S` too to have a consistent behaviour and less ambiguity.

> * If I want to suffix it `-stormi-test-build-for-I225-V`, I can.

You can already by selecting a full name. Leaving the filename as a caller choice decouples implementation from policy, which is generally a good idea.  Probably `create-installimg` should not use a default output filename anyway like `create-install-iso`, and both should refuse toclobber unless some `--force-overwrite` is given.

Reply:
change done, no default output filename any more

Reply:
`.treeinfo` content generation from branding has been added to TODO list already